components/NoteInputField.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+    <div
+        class="relative w-fit"
+    >
+        <div class="flex items-center justify-between w-96">
+            <label
+                class="mb-3.5 text-center text-primary-text text-2xl font-bold font-sans leading-normal"
+            >Moderator Notes</label>
+            <div
+                class="absolute top-0 right-0 mt-2 mr-2"
+            >
+                <button
+                    v-if="!isEditing"
+                    data-testid="note-edit-button"
+                    @click="startEditing"
+                >
+                    <PencilEdit class="w-4" />
+                </button>
+                <button
+                    v-else
```

Comment:
Usually we try to be more explicit about when to render something in this codebase. So even though this works I would suggest 
```suggestion
                    v-if="isEditing"
```